README.md

```diff
@@ -19,8 +19,8 @@ Sign up to [get started for free](https://app.revenuecat.com/signup).
 
 React Native Purchases is the client for the [RevenueCat](https://www.revenuecat.com/) subscription and purchase tracking system. It is an open source framework that provides a wrapper around `StoreKit`, `Google Play Billing` and the RevenueCat backend to make implementing in-app purchases in `React Native` easy.
 
-## Migrating from Purchases v3
-- See our [Migration guide](https://revenuecat-docs.netlify.app/documentation/revenuecat/v4_api_migration_guide)
```

Comment:
Should we keep this old section as well? (not sure how many people would still be using v3...)

Reply:
the V3 there is a terrible copy-paste bug, we don't actually have a migration guide from v3 of React-Native. That was just pointing to the V3 -> V4 migration guide on iOS